### DISCUSSION TITLE
ERROR! this task "ansible.buildin.import_tasks" has extra params, which...

### DISCUSSION THREAD

**Discussion by malawi92** [hifis-net/ansible-collection-toolkit#108] [Q&A] [answered]
ERROR! this task 'ansible.buildin.import_tasks' has extra params, which is only allowed in the following modules: include_tasks, raw, command, import_tasks, add_host, include, group_by, meta, include_role, win_shell, win_command, script, shell, import_role, set_fact, include_vars

The error appears to be in '/ansible/AG_Linux/roles/hifis.unattended_upgrades/tasks/main.yml': line 2, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

---
- name: "Import tasks from the unattended-upgrades playbook"
  ^ here

------------------------

I got the issue above. 

Ansible version 2.9.6
Python version 3.8.10

**Comment by Normo** [comment]
Dear @malawi92, thanks for reporting this!
The minimum supported Ansible version for this role is currently `2.12` ( see [meta/main.yml](https://github.com/hifis-net/ansible-role-unattended-upgrades/blob/v2.0.1/meta/main.yml#L8)). Could you please try again with a more recent Ansible version?

**Comment by tobiashuste** [comment, marked as answer]
Dear @malawi92, I think the reported message might be related to a typo.

It should be

```
ansible.builtin.import_tasks
```

instead of

> ansible.buil**d**in.import_tasks

Please mark as answer if this resolves your issue.
